### PR TITLE
[CPU] Proposal node migration on nGraph.

### PR DESCRIPTION
**Pull request by nshchego**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/nodes/proposal.cpp [hunk 1]
```diff
@@ -75,32 +80,46 @@ std::vector<float> generate_anchors(proposal_conf &conf) {
...
+            conf.scales = proposalAttrs.scale;
+            conf.ratios = proposalAttrs.ratio;
+            conf.normalize_ = proposalAttrs.normalize;
+            conf.clip_before_nms = proposalAttrs.clip_before_nms;
+            conf.clip_after_nms = proposalAttrs.clip_after_nms;
             conf.anchors_shape_0 = conf.ratios.size() * conf.scales.size();
 
-            std::string framework_ = layer->GetParamAsString("framework", "");
-            if (framework_ == "tensorflow") {
+            if (proposalAttrs.framework == "tensorflow") {
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/proposal.cpp:122]: We have to check supported range of values for `framework` in `isSupportedOperation`

File: inference-engine/src/mkldnn_plugin/nodes/proposal_onnx.cpp [hunk 2]
```diff
@@ -282,25 +286,44 @@ class ONNXCustomProposalImpl : public ExtLayerBase {
...
+            auto proposalAttrs = proposalOp->get_attrs();
 
-            min_size_ = layer->GetParamAsFloat("min_size");
-            nms_thresh_ = layer->GetParamAsFloat("nms_threshold");
-            pre_nms_topn_ = layer->GetParamAsInt("pre_nms_count");
-            post_nms_topn_ = layer->GetParamAsInt("post_nms_count");
+            min_size_ = proposalAttrs.min_size;
+//            nms_thresh_ = proposalAttrs.nms_threshold;
+//            pre_nms_topn_ = proposalAttrs.pre_nms_count;
+//            post_nms_topn_ = proposalAttrs.post_nms_count;
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/proposal_onnx.cpp:315]: Seems like unused. If so please delete

File: inference-engine/src/mkldnn_plugin/nodes/proposal.cpp (hunk 1 above)
Comment (nshchego) [reply inference-engine/src/mkldnn_plugin/nodes/proposal.cpp:127]: Fixed

File: inference-engine/src/mkldnn_plugin/nodes/proposal_onnx.cpp (hunk 2 above)
Comment (nshchego) [reply inference-engine/src/mkldnn_plugin/nodes/proposal_onnx.cpp]: Actually they are used. There was required another operation: ExperimentalDetectronGenerateProposalsSingleImage. Fixed.

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**